docker/Dockerfile

```diff
@@ -6,3 +6,6 @@ RUN cd /usr/src/node-red/node_modules/node-red/nodes/core; \
     awk '{system("mv "$1" "$1".ign")}';
 RUN cd /usr/src/node-red/node_modules;  ls -1 | grep 'node-red-node' | xargs rm -rf
 COPY node_modules /data/node_modules
+USER root:root
```

Comment:
by default the non-root user node-red is used, since root user is only needed because things are copied with root ownership, wouldn't it be best to chmod them and keep the non-root user as the effective user for the container?